hypnos/include/event_handler.h

```diff
@@ -8,12 +8,14 @@
 #define EVENT_HANDLER__H
 
 #include <drivers/gpio.h>
+#include <drivers/sensor.h>
 
 void event_handler_init(void);
 void backlight_off_interrupt_handler(struct k_timer *);
 void battery_percentage_interrupt_handler(struct k_timer *);
 void battery_callback_charging(struct device*, struct gpio_callback *, u32_t);
 void clock_callback_tick(struct k_timer *);
 void button_callback(struct device *, struct gpio_callback *, u32_t);
+void touch_callback_tap(struct device *, struct sensor_trigger *);
```

Comment:
Button and battery callback both have a u32_t parameter, should touch also have that?

Reply:
Maybe, yes... Jan's example code doesn't have it, but in the driver code I found this:

```
static void cst816s_gpio_callback(struct device *dev,
		struct gpio_callback *cb, u32_t pins)
{
	struct cst816s_data *drv_data =
		CONTAINER_OF(cb, struct cst816s_data, gpio_cb);

	ARG_UNUSED(pins);

	gpio_pin_disable_callback(dev, CONFIG_CST816S_GPIO_PIN_NUM);

#if defined(CONFIG_CST816S_TRIGGER_OWN_THREAD)
	k_sem_give(&drv_data->gpio_sem);
#elif defined(CONFIG_CST816S_TRIGGER_GLOBAL_THREAD)
	k_work_submit(&drv_data->work);
#endif
}
```

Reply:
Hmm, it seems that it should be included. The compiler complained anyway and without the third argument all is fine.